bin/ebash

```diff
@@ -119,13 +117,29 @@ else
 
     # Use all arguments to ebash as a command to execute
     if [[ ${#@} -gt 0 ]] ; then
-        # Run the command, but pretend to be inside a try so that die doesn't
-        # print an ugly stack trace.
+
+        # If we were given the path to another script run it inside ebash environment
+        if [[ ${#@} -eq 1 && -e "${1}" ]]; then
+
+            export EBASH EBASH_HOME
+
+            SOURCE="${1}"
+            SCRIPT=$(mktemp --tmpdir ebash-script-XXXXXX)
+            trap_add "rm ${SCRIPT}"
```

Comment:
Do you want `rm -f` so it doesn't fail if the script goes away for its own reasons?

Reply:
Good suggestion, I added that!